python/ray/worker.py

```diff
@@ -1080,6 +1080,48 @@ def default_dict_custom_deserializer(obj):
         custom_serializer=default_dict_custom_serializer,
         custom_deserializer=default_dict_custom_deserializer)
 
+    def _serialize_pandas_series(s):
+        import pandas as pd
+        # TODO: serializing Series without extra copy
+        serialized = serialize_pandas(pd.DataFrame({s.name: s}))
+        return {
+            'type': 'Series',
+            'data': serialized.to_pybytes()
+         }
+
+
```

Comment:
too much white space between these functions, should be one line, run `flake8 worker.py`.